src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public/tests/dashboard_container.test.tsx

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
is this file still needed?  Looks like you kept the original one in it's original location, this is now the only file in the `tests` folder (I prefer test files alongside the file they are testing, I believe that's the most common pattern).

Reply:
This file is need. Original one has simple unit tests that test just the component, hence it is kept next to the component like all other unit tests.

The test in this file spins up the whole Embeddables API using

```ts
embeddablePluginMock.createInstance();
```

It is basically an integration test, hence I put it on the top level.